charms/kserve-controller/src/templates/serving_runtimes_manifests.yaml.j2

```diff
@@ -3,6 +3,9 @@ kind: ClusterServingRuntime
 metadata:
   name: kserve-lgbserver
 spec:
+  annotations:
```

Comment:
do you know how these work?  I think these might be configurations for how prometheus discovers and monitors things, but we get this from the cos stack instead.  I might be misunderstanding it though

Reply:
So according to this https://kserve.github.io/website/latest/modelserving/observability/prometheus_metrics/ its a specification where prometheus metrics for individual deployments are exposed. We need to specify this so later prometheus can gather those. I think its needed otherwise the deployments wont be exposing metrics.

Reply:
I'm not 100% sure if we should leave this, but I'm 100% sure it wont break anything :)  So lgtm